File: Infrastructure-Module/src/main/kotlin/com/bamyanggang/infrastructuremodule/persistence/config/PersistenceConfig.java
package com.bamyanggang.infrastructuremodule.persistence.config;

import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@Configuration
@EntityScan(basePackages = {"com.bamyanggang.infrastructuremodule.persistence"})
@EnableJpaRepositories(basePackages = {"com.bamyanggang.infrastructuremodule.persistence"})
@EnableJpaAuditing
public class PersistenceConfig {
}

File: Infrastructure-Module/src/main/kotlin/com/bamyanggang/infrastructuremodule/persistence/config/PersistenceConfig.kt
package com.bamyanggang.infrastructuremodule.persistence.config

import org.springframework.boot.autoconfigure.domain.EntityScan
import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@Configuration
@ComponentScan(basePackages = ["com.bamyanggang.infrastructuremodule.persistence"])
@EntityScan(basePackages = ["com.bamyanggang.infrastructuremodule.persistence"])
@EnableJpaRepositories(basePackages = ["com.bamyanggang.infrastructuremodule.persistence"])
class PersistenceConfig
